kuma/users/views.py

```diff
@@ -745,6 +764,36 @@ def recover(request, uidb64=None, token=None):
     return render(request, 'users/recover_failed.html')
 
 
+@require_POST
+def create_stripe_subscription(request):
+    user = request.user
+
+    error_message = None
+    try:
+        customer = None
+        if not user.stripe_customer_id:
+            customer = stripe.Customer.create(
+                email=request.POST.get('stripe_email', ''),
+                source=request.POST.get('stripe_token', ''),
+            )
+            user.stripe_customer_id = customer.id
+            user.save()
+        else:
+            customer = stripe.Customer.retrieve(user.stripe_customer_id)
+
+        if retrieve_stripe_subscription(customer) is None:
+            stripe.Subscription.create(
+                customer=customer.id,
+                items=[{
+                    'plan': settings.STRIPE_PLAN_ID,
+                }]
+            )
+    except stripe.error.StripeError as e:
+        error_message = e.message
```

Comment:
As [demonstrated here](https://github.com/mdn/kuma/pull/6269#issuecomment-569982844) this isn't right. 
A `StripeError` doesn't necessarily have a message. For example, a `AuthenticationError` exception is a subclass of `StripError` but no `.message`.